modules/sql/src/main/scala/enum/GnirsEnums.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+package enum
+
+import doobie._, doobie.implicits._
+import shapeless.record._
+
+object GnirsEnums {
+  import Angle.Arcseconds
+
+  val enums: List[ConnectionIO[EnumDef]] =
+    List(
+
+      EnumDef.fromQuery("GnirsCrossDispersed", "GNRIS Cross Dispersed") {
+        type E = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'pixelScale -> BigDecimal`.T
+        sql"""SELECT id, id tag, short_name, long_name, pixel_scale FROM e_gnirs_cross_dispersed""".query[(String, E)]
+      },
+
+      EnumDef.fromQuery("GnirsFpuSlit", "GNRIS FPU Slit") {
+        type E = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'slitWidth -> Arcseconds`.T
+        sql"""SELECT id, id tag, short_name, long_name, pixel_scale FROM e_gnirs_cross_dispersed""".query[(String, E)]
```

Comment:
Oops, this is reading from the wrong table.  Should be `e_gnirs_fpu_slit` or whatever it is called.